karma.py

```diff
@@ -1,4 +1,5 @@
 from brutal.core.plugin import BotPlugin, cmd, match
+from brutal.core.utils import change_cmd
```

Comment:
@pepol for instance storage or help. Tough world, we should just assume that the `brutal` project is just dead, long live `brutal-ng`!